KAFKA_VERSION_SUPPORT.md

```diff
@@ -0,0 +1,7 @@
+# Supported Apache Kafka versions
+
+It is not feasible to maintain support for all available Apache Kafka versions.
+The following rules describe our plans for Kafka versions supported in Strimzi:
+
+* Support at least the last two minor versions of Apache Kafka. _For example: Support for Kafka 2.2.x could be removed when support for 2.4.x is added._
+* Support at least one common Kafka release in two consecutive minor Strimzi releases to allow smooth upgrades between Strimzi and Kafka versions. _For example: when Strimzi 0.13.x supports Kafka 2.2.0 and 2.3.0, Strimzi 0.14.x has to provide support for 2.3.0 as well._
```

Comment:
As a maintainer I have no complaint about what's described here. I think it might be worth explaining how it would work across major Kafka version changes because `3.x` will happen sooner or later (I think the same "two major/minor versions back" policy would work).

I'm not sure having this policy buried in some file in the repo is really sufficiently obvious for users. It might be better on the website close to the docs (but probably not in the docs, since it's not a functional thing).

Reply:
I changed the text to refer to `major/minor` There is probably no reason to not do that, although major versions are always a lot of uncertainty.

As for the placement, I would maybe start here and see hwere else it could be added in the future. I'm not exactly sure where on the site I would add this TBH.

Reply:
It might be easier once the site is restructured.